pkgs/by-name/py/pyfa/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  appimageTools,
+  fetchurl,
+}:
+let
+  version = "2.60.1";
+in
+appimageTools.wrapType1 {
+  inherit version;
+  pname = "pyfa";
+
+  src = fetchurl {
+    url = "https://github.com/pyfa-org/Pyfa/releases/download/v${version}/pyfa-v${version}-linux.AppImage";
+    name = "pyfa-${version}";
+    hash = "sha256-+nyIG6pweIgC6ZiWJaO4E2Fp0/zP+DhQrUWzHnIBQXk=";
+  };
+
+  meta = {
```

Comment:
Since you are not building this from sources, set `sourceProvenance` and `platforms` accordingly.

Reply:
can you check if that is correct?